projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -110,12 +113,14 @@ const ProductCard = props => {
 	const isActive = status === PRODUCT_STATUSES.ACTIVE;
 	const isError = status === PRODUCT_STATUSES.ERROR;
 	const isInactive = status === PRODUCT_STATUSES.INACTIVE;
-	const isAbsent = status === PRODUCT_STATUSES.ABSENT;
+	const isAbsent = status === PRODUCT_STATUSES.ABSENT || status === PRODUCT_STATUSES.NEEDS_PURCHASE;
+	const isPurchaseRequired = status === PRODUCT_STATUSES.NEEDS_PURCHASE;
 	const flagLabel = PRODUCT_STATUSES_LABELS[ status ];
 	const canDeactivate = ( isActive || isError ) && admin;
 
 	const containerClassName = classNames( styles.container, {
 		[ styles.plugin_absent ]: isAbsent,
+		[ styles[ 'is-purchase-required' ] ]: isPurchaseRequired,
```

Comment:
Seems like this styles is not used or defined anywhere else

Reply:
True... I want to leave a hint that we may change the UI based on it. Let me remove it.

Reply:
I've applied the same styles that when the plugin is absent [here](https://github.com/Automattic/jetpack/pull/22687/commits/ceaaa26294c66a71a7c71afa501fd3df8aa4377f).